grammar.js

```diff
@@ -202,15 +219,19 @@ module.exports = grammar({
 
     change: ($) =>
       seq(
-        field("kind", CHANGE),
+        field("kind", choice("new file", "modified", "renamed", "deleted")),
         ":",
         $.path,
         optional(seq(token(prec(PREC.PATH_SEPARATOR_ARROW, "->")), $.path))
       ),
 
     commit: ($) => /[a-f0-9]{7,40}/,
-
-    _word: ($) => token(prec(PREC.NONSENSE, /\S+/)),
+    _non_punctuated_word: ($) => token(prec(0, /[-\w]+/)),
```

Comment:
Done. I've also separated the PREC constant into a `LEXICAL_PRECEDENCE` and `PARSE_PRECEDENCE` for additional clarity. 

For future consideration: the `PARSE_PRECEDENCE` constant and all references to it might be replaced with a `precedences` array in the `grammar({...})` object; see https://tree-sitter.github.io/tree-sitter/creating-parsers#the-grammar-dsl in the "precedences" bullet point right before the next section.